src/iwork_common.h

```diff
@@ -12,7 +12,7 @@
 #define SALTLEN                 16
 #define IVLEN                   16
 #define BLOBLEN                 64
-#define FORMAT_NAME             "Apple iWork '09 / '13 / '14"
+#define FORMAT_NAME             "Apple iWork '09 to 2018"
```

Comment:
Using different notions for 2009 and 2018 looks wierd. Was the software using different notions? Anyway perhaps you should just leave out the years, until we see some newer version with a different scheme? So either just `"Apple iWork"` or `"Apple iWork '09 or newer"`

Reply:
Yes, it looks weird. Yes, the software has used different styles over the years. See https://en.wikipedia.org/wiki/IWork#Versions for the details.

I am happy with `Apple iWork '09 or newer` suggestion, thanks!